test/08_manylinuxXXXX_only/cibuildwheel_test.py

```diff
@@ -12,9 +14,11 @@ def test():
     # rather than when dynamically loading the Python
     actual_wheels = utils.cibuildwheel_run(project_dir, add_env={
         'CIBW_ENVIRONMENT': 'CFLAGS="$CFLAGS -Werror=implicit-function-declaration"',
+        'CIBW_SKIP': 'cp27*',  # not available on manylinux2014
```

Comment:
Agreed. I'm fine dropping 2.7 for this specific test.

Reply:
Wait, actually... I agree we don't need to test 2.7 in this particular test, but ... do we force the user to put `CIBW_SKIP` to `cp27*` when using `manylinux2014`? Because running with only `CIBW_MANYLINUX_..._IMAGE=manylinux2014` will result in an error, now.

Reply:
I really don't want to clutter `linux.py` with specific code. I was thinking about wether documentation only or adding a check for python existence (not specific to manylinux2014). Since it's opt-in, I did not mind not changing `linux.py`. Documentation shall be updated though.

Reply:
> Documentation shall be updated though.

OK, good point. I guess we'll soon enough be dropping 2.7 anyway? We should maybe start thinking about how to phase out support?

Reply:
Documentation updated. For phasing-out support of 2.7, we should start thinking about how to do it but this PR is probably not the place.

Reply:
Could this change to only skip 2.7 when `manylinux_image == 'manylinux2014'`? While we're still supporting 2.7 we should check it still works on manylinux1. Want to be sure that our last 2.7 release is good!

Reply:
Eh, I also thought so for a second, but then in principle, 2.7 is tested in the other tests already? So for the `manylinuxXXXX_only` tests, I'm not sure it's very urgent.

Reply:
I don't think it's covered on the manylinux1 image, but I might be wrong?

Reply:
> I don't think it's covered on the manylinux1 image, but I might be wrong?

Hmmm, true.

Reply:
restored tests with `cp27` on `manylinux1` and `manylinux2010`